songpal/device.py

```diff
@@ -129,7 +130,7 @@ async def request_supported_methods(self):
         """Return JSON formatted supported API."""
         return await self.create_post_request("getSupportedApiInfo")
 
-    async def get_supported_methods(self):
+    async def get_supported_methods(self, default_latest: bool = False):
```

Comment:
```suggestion
    async def get_supported_methods(self, *, default_latest: bool = False):
```
Let's mark this as kwarg-only.